network-boot-lab.md

```diff
@@ -4,17 +4,17 @@
 
 Complete the instructions described in this lab.
 
-The student must send:
-1. A `zip file` containing the final folder after completing this tutorial and just removing the `data` folder.
+The student must send (--> where?):
```

Comment:
They enable a component with a dropdown to upload the files

Reply:
The idea is that they create a section like this 
<img width="1680" alt="Screen Shot 2022-01-03 at 13 57 12" src="https://user-images.githubusercontent.com/28828796/147974303-7eca54bf-f632-43f5-908e-3145c682095b.png">